sql/core/pom.xml

```diff
@@ -130,6 +130,12 @@
       <scope>test</scope>
     </dependency>
     <dependency>
+      <groupId>org.apache.avro</groupId>
```

Comment:
Is this issue only a test dependency? I see avro.version declared in the top level pom.xml